modules/standard/IO.chpl

```diff
@@ -2984,6 +2992,7 @@ private inline proc _read_one_internal(_channel_internal:qio_channel_ptr_t,
                                        ref x:?t,
                                        loc:locale):syserr where _isIoPrimitiveTypeOrNewline(t) {
   var e:syserr = ENOERR;
+
```

Comment:
You can & should remove these spacing changes (since they were from code you added and then removed)